config/owasp/suppressions.xml

```diff
@@ -7,36 +7,4 @@
     <packageUrl regex="true">^pkg:maven*.*$</packageUrl>
     <cve>CVE-2018-1258</cve>
   </suppress>
-  <suppress until="2021-07-05">
-    <notes><![CDATA[
```

Comment:
just reminder to remove these CVEs in confluence if not done.

Reply:
it is being taken care already by @pksaxena01